tests/fixtures/phpcs/DrupalClass.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\testmodule;
+
+use Drupal\BlockBase;
+
+/**
+ * Some service.
+ */
+class DrupalClass extends BlockBase {
+
+  /**
+   * Using \Drupal here but it should be injected instead.
+   *
+   * @deprecated Test.
+   */
+  public function test() {
+    return true;
```

Comment:
This error is not needed to be tested.

Reply:
This is simply recreating a normal scenario where you have some errors.
It could be only the deprecated but we can also have others and probably we'll want to add more, not to remove.

Reply:
ok